erpnext_com/www/docs/user/manual/en/using-erpnext/access-log.md

```diff
@@ -10,8 +10,9 @@ metatags:
 
 # Access Log
 
+> Introduced in Version 13
```

Comment:
rearranged with reference to [this](https://erpnext.com/docs/user/manual/en/setting-up/automation/milestone-tracker)